programs/editor/Tools.js

```diff
@@ -183,6 +186,36 @@ define("webodf/editor/Tools", [
                     saveButton.placeAt(toolbar);
                 }
 
+                // SaveAs
+                if (saveAsOdtFile) {
```

Comment:
Yes, proper extensibility would be good to have. But for now lower priority than the Closure port (for me). Once that port is done, extensibility (along plugin support) should be gone for.

Besides that general comment... did I see a ship-it hidden between the lines? :angel: